models.py

```diff
@@ -550,6 +550,12 @@ def _format_direction(view, context, model, name):
     elif model.direction == 1:
         return Markup('in')
 
+def get_bank_account_number(view, context, model, name):
```

Comment:
why not put this function next to `_format_address_column` and use a similar name like `_format_bank_account`

you should be able to just use `model.banks.first()`

Reply:
made the change.

Reply:
deleted that function.